zeppelin-plugins/launcher/k8s-standard/src/main/java/org/apache/zeppelin/interpreter/launcher/K8sRemoteInterpreterProcess.java

```diff
@@ -240,30 +238,31 @@ void apply(File path, boolean delete) throws IOException {
       }
 
       for (File f : files) {
-        apply(f, delete);
+        apply(f, delete, templateProperties);
       }
     } else if (path.isFile()) {
-      LOGGER.info("Apply {}", path.getAbsolutePath());
       K8sSpecTemplate specTemplate = new K8sSpecTemplate();
-      specTemplate.loadProperties(getTemplateBindings());
-
-      String spec = specTemplate.render(path);
+      String template = specTemplate.render(path);
+      specTemplate.loadProperties(templateProperties);
```

Comment:
Looks like `loadProperties()` should be placed before `render()`

Reply:
You're absolutely right. I don't know how this critical error occurred (maybe in the last cleanup before committing and after my tests).

Reply:
I also didn't recognize this one earlier :) Anyway, I tested this branch and everything works well on my environment as well. LGTM. 

(I think you can [configure merge script](https://github.com/apache/zeppelin/blob/master/dev/merge_zeppelin_pr.py#L41-L45) and merge this PR)

Reply:
Hi @Leemoonsoo,
thanks for testing. Before merging, I will test if Zeppelin can be started locally and connect to an interpreter in the K8 cluster via port forwarding.